test/bankAccounts.js

```diff
@@ -60,8 +60,8 @@ describe('bank accounts', () => {
       Lob.bankAccounts.list((err, res) => {
         expect(res.object).to.eql('list');
         expect(res.data).to.be.instanceof(Array);
-        expect(res.data.length).to.eql(10);
-        expect(res.count).to.eql(10);
+        expect(res.data.length <= 10).to.be.true;
```

Comment:
nit: Can you please use `expect(res..data.length).to.be.at.most(10)` instead? It leaves a more helpful error message on failure.

[Docs](https://www.chaijs.com/api/bdd/#method_most)

Reply:
Here and elsewhere.

Reply:
Sure thing. I'll make that fix real quick!

Reply:
Updated in e05a00f